### PR TITLE
Elaborates on DBaaS slugs and the Windows CLI

### PR DESCRIPTION
**Pull request by yGabriel**
## Description and Details
Resolves PDOCS-1192 by elaborating on DBaaS slugs and adding a paragraph on following `curl` examples with the Windows CLI.

## Before approving, please test or review...
Verify technical wording and style.

## Checklist

- Site documentation and test coverage
    - [X] This PR does not require additional site documentation or test coverage, or
    - [ ] This PR includes necessary site documentation or test coverage.
- Blockers
    - [X] This PR can be merged as soon as it is approved, or
    - [ ] This PR is complete but should not be merged when it is approved; I have added the `DO-NOT-MERGE` label and described when it can be merged above, or
    - [ ] This PR is incomplete and should not be merged; I have added the `WIP` label and tagged a specific reviewer for an in-progress review.

### CODE REVIEW

**Comment by danaelhe** [comment]
looks great! Will merge after code freeze.

File: specification/DigitalOcean-public.v2.yaml [hunk 1]
```diff
@@ -149,22 +149,37 @@ tags:
...
+
+      Slug Size    | RAM   | CPU     | Storage |
+      -------------|-------|---------|---------|
+      1vcpu-1gb    | 1 GB  | 1 vCPU  | 10 GB   |
+      1vcpu-2gb    | 2 GB  | 1 vCPU  | 25 GB   |
+      2vcpu-4gb    | 4 GB  | 2 vCPU  | 38 GB   |
+      4vcpu-8gb    | 8 GB  | 4 vCPU  | 115 GB  |
+      6vcpu-16gb   | 16 GB | 6 vCPU  | 270 GB  |
+      8vcpu-32gb   | 32 GB | 8 vCPU  | 580 GB  |
+      16vcpu-64gb  | 64 GB | 16 vCPU | 1.12 TB |
```
Comment (andrewsomething) [inline specification/DigitalOcean-public.v2.yaml:178]: Are all of these actually available in the `db-s`, `gd`, and `so1_5` variants?  For example, looking in the control panel, I see the smallest "General Purpose Node" available for me is 8 GB RAM and 2 vCPUs which doesn't match any of these. How many size slugs are available in total? Would it make sense to explicitly list them all?

Comment (yGabriel) [reply specification/DigitalOcean-public.v2.yaml:178]: Hi, Andrew! You're absolutely right; the node plans are different based on machine type. I also found that there are exceptions based on database engine. For example, MongoDB offers greater disk size (and offers standby nodes with the smallest node size/machine type), and Redis has a slightly different node plan description.

Without those exceptions, there are around 19 permutations to potentially list. I'll take a look at this today and notify you for another review.

**Comment by andrewsomething** [comment]
!deploy

File: specification/description.yml [hunk 2]
```diff
@@ -260,6 +260,12 @@ introduction: |
   Throughout this document, some example API requests will be given using the
   `curl` command. This will allow us to demonstrate the various endpoints in a
   simple, textual format.
+    
+  These examples assume that you are using a Linux or macOS command line. To run
+  these `curl` commands in a Windows command line, you must change all single
+  quotes (`''`) in the command to double quotes (`""`) and escape nested quotes
+  with a slash (`/`). For example, instead of `'{"size":"db-s-4vcpu-8gb", "num_nodes":3}'`,
+  use `"{\"size\":\"db-s-4vcpu-8gb\", \"num_nodes\":3}"`.
```
Comment (andrewsomething) [inline specification/description.yml:268]: I'm not on a Windows machine, but I'm curious if using PowerShell over the command prompt is an option here. I know it supports single quotes, but don't know if curl is available for it or not. If it is, that might be easier than having to rewrite the commands.

Comment (andrewsomething) [reply specification/description.yml:268]: This doesn't need to be blocking if you don't know off hand. We can always come back to this. Honestly, I feel like this whole curl examples section could use a bit of a rewrite.

Comment (yGabriel) [reply specification/description.yml]: Just finished following up on this! Have up-to-date solutions for cmd.exe, PowerShell, and WSL methods with the help of @ElanHasson.

**Comment by yGabriel** [comment]
!deploy

**Comment by andrewsomething** [comment]
!deploy

**Review by andrewsomething — APPROVED** [review]
:+1:  Thanks @yGabriel!

**Pull request merged**